infrastructure/core/data.tf

```diff
@@ -14,8 +14,19 @@
 
 data "azurerm_client_config" "current" {}
 
-data "azurerm_container_registry" "devcontainer" {
+data "http" "local_ip" {
+  count = var.tf_in_automation ? 0 : 1
+  url   = "https://api64.ipify.org"
+}
+
+data "azurerm_virtual_network" "ci" {
```

Comment:
Yeah I was torn between ci and bootstrap - I decided ci just because that's the exported secrets prefix we have for all the bootstrap resources, plus I think maybe people are just a bit more familiar with the term ci - but not fussed either way if you feel strongly?

Reply:
You kind of talk about that in README.md indirectly so I think that's fine